docs/advanced/singlenodeclusters.md

```diff
@@ -1,33 +1,60 @@
 
 Single-node clusters support most Harvester features, including the creation of RKE2 clusters and node upgrades (with some limitations). However, this deployment type has the following key disadvantages:
 
 - No high availability: Errors and updates that require rebooting of the node cause downtime to running VMs.
-- No multi-replica support: Only one replica is created for each volume in Longhorn.
 - No live migration and zero-downtime support during upgrades.
 
 ## Prerequisites
 
-Before you begin deploying your single-node cluster, ensure that the following requirements are addressed.
+Before you begin deploying your single-node cluster, ensure that the following requirements are met.
 
-- Hardware: [Use server-class hardware](https://docs.harvesterhci.io/v1.3/install/requirements#hardware-requirements) with sufficient resources to run Harvester and a production workload. Laptops and nested virtualization are not supported.
-- Network: [Configure ports](https://docs.harvesterhci.io/v1.3/install/requirements#port-requirements-for-harvester-nodes) based on the type of traffic to be transmitted among VMs.
-- StorageClass: [Create a new default StorageClass](https://docs.harvesterhci.io/v1.3/advanced/storageclass#creating-a-storageclass) with the **Number of Replicas** parameter set to "1". This ensures that only one replica is created for each volume in Longhorn.
+- Hardware: [Use server-class hardware](../install/requirements.md#hardware-requirements) with sufficient resources to run Harvester and a production workload. Laptops and nested virtualization are not supported.
+- Network: [Configure ports](../install/requirements.md#port-requirements-for-harvester-nodes) based on the type of traffic to be transmitted among VMs.
 
-  :::info important
-  The default StorageClass `harvester-longhorn` has a replica count value of `3` for high availability. If you use this StorageClass to create volumes for your single-node cluster, Longhorn is unable to create the configured number of replicas. This results in volumes being marked as "Degraded" on the Longhorn UI.
-  :::
+## Replica Count of the Default StorageClass 
+
+Harvester uses StorageClasses to describe how Longhorn must provision volumes. Each StorageClass has a parameter that defines the number of replicas to be created for each volume. 
+
+The default StorageClass `harvester-longhorn` has a replica count value of **3** for high availability. If you use `harvester-longhorn` in your single-node cluster, Longhorn is unable to create the default number of replicas, and volumes are marked as *Degraded* on the [embedded Longhorn UI](../troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards). 
```

Comment:
We can observe `Degraded` on the **Volumes** page directly. No need to enter LH.
![image](https://github.com/user-attachments/assets/1623c0b9-6f46-406e-a3f0-776d72bec0b1)

Reply:
Thank you. I have updated the doc.